sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/DefaultServiceSpecTest.java

```diff
@@ -83,7 +83,8 @@ public void validSimple() throws Exception {
         File file = new File(classLoader.getResource("valid-simple.yml").getFile());
         DefaultServiceSpec serviceSpec = DefaultServiceSpec.newGenerator(RawServiceSpec.newBuilder(file).build(), flags).build();
         Assert.assertNotNull(serviceSpec);
-        Assert.assertFalse(DefaultService.serviceSpecRequestsGpuResources(serviceSpec));
```

Comment:
We do: https://github.com/mesosphere/dcos-commons/blob/8585d5a787bd8d8d9e19627f98926f17ccfd3eb6/sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/DefaultServiceSpecTest.java#L91